### ISSUE TITLE
Upload card view arrow points in random direction

### ISSUE THREAD

**Issue by domdomegg** [commons-app/apps-android-commons#2205]
**Summary:** 

The arrow to show and hide the box to write the title and description in the new upload flow seems to point in random directions after the first upload.

**Steps to reproduce:**

How can we reproduce the issue?
1. Open the app
2. Start upload flow
3. Exit from upload flow
4. Start upload flow again

**Screen-shots:** 

See https://giphy.com/gifs/2UEYQEXs9z51pNI5u5

**Comment by neslihanturan** [comment]
Thanks for reporting @domdomegg , funny issue though:D If you don't plan to work on this issue, can we promote this for beginners?

**Comment by domdomegg** [comment]
Have added beginner friendly tag. Have not looked into it at all though so it might be difficult, will remove if it turns out to be.

**Comment by NicholasWeiner** [comment]
I am a beginner so I will take on this issue.

**Comment by domdomegg** [comment]
Assigned to @NicholasWeiner

**Comment by NicholasWeiner** [comment]
@domdomegg I can't seem to replicate the issue on my device? What Android version is your phone using?

**Comment by domdomegg** [comment]
I reproduced it on the `Galaxy Nexus (simulator)` at API level `28`. Will post full details in a bit.

**Comment by domdomegg** [comment]
Details:

```
Name: Galaxy_Nexus_API_28
CPU/ABI: Google APIs Intel Atom (x86)
Path: ~/.android/avd/Galaxy_Nexus_API_28.avd
Target: google_apis [Google APIs] (API level 28)
Skin: 720x1280
SD Card: 512 MB
fastboot.chosenSnapshotFile: 
runtime.network.speed: full
hw.accelerometer: yes
hw.device.name: Galaxy Nexus
hw.lcd.width: 720
hw.initialOrientation: Portrait
image.androidVersion.api: 28
tag.id: google_apis
hw.mainKeys: no
hw.camera.front: emulated
avd.ini.displayname: Galaxy Nexus API 28
hw.gpu.mode: auto
hw.ramSize: 1024
PlayStore.enabled: false
fastboot.forceColdBoot: no
hw.cpu.ncore: 4
hw.keyboard: yes
hw.sensors.proximity: yes
hw.dPad: no
hw.lcd.height: 1280
vm.heapSize: 80
skin.dynamic: yes
hw.device.manufacturer: Google
hw.gps: yes
skin.path.backup: _no_skin
hw.audioInput: yes
image.sysdir.1: system-images/android-28/google_apis/x86/
showDeviceFrame: no
hw.camera.back: virtualscene
AvdId: Galaxy_Nexus_API_28
hw.lcd.density: 320
hw.arc: false
hw.device.hash2: MD5:5c288d27461585ecc73a535555e7cf61
fastboot.forceChosenSnapshotBoot: no
fastboot.forceFastBoot: yes
hw.trackBall: no
hw.battery: yes
hw.sdCard: yes
tag.display: Google APIs
runtime.network.latency: none
disk.dataPartition.size: 2G
hw.sensors.orientation: yes
avd.ini.encoding: UTF-8
hw.gpu.enabled: yes
```

Recreated on master @ b447612cc

![screenshot_1545622681](https://user-images.githubusercontent.com/4953590/50390294-6d4a0100-072d-11e9-8014-18f2c3278ed3.png)
![screenshot_1545622686](https://user-images.githubusercontent.com/4953590/50390295-6d4a0100-072d-11e9-8e6e-5c1e32551456.png)
![screenshot_1545622719](https://user-images.githubusercontent.com/4953590/50390296-6d4a0100-072d-11e9-8e51-5bb38feec463.png)

**Comment by mrpeny** [comment]
Hi, I am planning to work on this issue. Are you still working on it @NicholasWeiner? Please let me know if you have discovered something that would be useful for further testing and fixing! Thank you

**Comment by NicholasWeiner** [comment]
Hi Bence,
The issue is all yours. I can no longer work on it due to having a busy
schedule for the next few months.

**Comment by mrpeny** [comment]
OK. Thank you. @domdomegg, please assign this issue to me if you agree!

**Comment by domdomegg** [comment]
Assigned to @mrpeny

**Comment by domdomegg** [comment]
Also @mrpeny It looks like you might have a fix in 762c210. I'd be happy to review a PR consisting of that change if you request a review from me :)

**Comment by mrpeny** [comment]
Thanks @domdomegg, I made the PR. Please let me know if something is not correct! I have read the documentation but I am new to the open source world so I might have missed something.
